tests/test_plugin_loader.py

```
@@ -739,5 +740,269 @@ def test_collect_requirements_empty_file(self):
+        self.addCleanup(lambda: shutil.rmtree(self.temp_dir, ignore_errors=True))
+
+        # Reset plugin loader state
+        import mmrelay.plugin_loader as pl
+
+        pl.sorted_active_plugins = []
+        pl.plugins_loaded = False
+
+    @patch("mmrelay.plugin_loader._clean_python_cache")
+    @patch("mmrelay.plugin_loader.load_plugins_from_directory")
+    def test_load_plugins_calls_cache_cleaning(
+        self, mock_load_plugins, mock_clean_cache
```

Ruff (0.14.3)

Unused method argument: `mock_load_plugins`

(ARG002)